### PR TITLE
🌟  feat: add action for tagging

### PR DESCRIPTION
**Pull request by nandiheath**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] The code passes all tests and able to build

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ..., provide screenshots if necessary)
Add the github action for tagging. Which will do the following

- when pushed to `main` branch, push `latest` and also `{SHA}` to dockerhub
- when it is tagged on github, it will try to pull the `image:${SHA}` and tag it with `image:${tag}`

BLOCKED by https://github.com/kintoproj/kinto-github-actions/pull/3

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -19,13 +19,14 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-
+      - name: Set env
+        run: echo "TAGS=kintohub/kinto-dashboard:${GITHUB_SHA},kintohub/kinto-dashboard:latest" >> $GITHUB_ENV
```
Comment (bakayolo) [inline .github/workflows/push.yml:23]: can u explain what this is doing?

Comment (nandiheath) [reply .github/workflows/push.yml:23]: sending the tags (latest + sha) to the next step.
somehow it was not woking if I wrote like this.
```
    tags: "kintohub/kinto-dashboard:${GITHUB_SHA},kintohub/kinto-dashboard:latest"
```

Comment (bakayolo) [reply .github/workflows/push.yml:23]: Gotcha

**Review by bakayolo — APPROVED** [review]
LGTM

**Pull request merged**